pkg/api/v1alpha1/cluster_types.go

```diff
@@ -34,6 +34,7 @@ type ClusterSpec struct {
 	// +kubebuilder:validation:Optional
 	ExternalEtcdConfiguration *ExternalEtcdConfiguration `json:"externalEtcdConfiguration,omitempty"`
 	ProxyConfiguration        *ProxyConfiguration        `json:"proxyConfiguration,omitempty"`
+	ECRMirror                 *ECRMirror                 `json:"ecrMirror,omitempty"`
```

Comment:
I have named it that because are only adding mirrors for ECR. My understanding is that if users wants to pull other images after the cluster is created, they can do something like `docker pull <local-registry>/some-image`